pages/content/posts/web-api-page-titles/index.md

```diff
@@ -0,0 +1,46 @@
+---
+author: "Will Bamberg"
+title: "The great renaming of Web/API"
+date: "2023-04-11"
+tags: ["projects"]
+ShowToc: false
+ShowBreadCrumbs: false
+---
```

Comment:
Can you add an `excerpt` front-matter? So that https://deploy-preview-71--beautiful-dieffenbachia-e315ce.netlify.app/content/posts/ isn't empty.

Reply:
I have had a go at this although it isn't very inspired :).